Libplanet.Explorer/Queries/Query.cs

```diff
@@ -51,6 +52,12 @@ public Query(BlockChain<T> chain, IStore store)
 
             while (limit is null || limit > 0)
             {
+                if (miner != null && miner == block.Miner)
+                {
+                    limit--;
+                    yield return block;
+                }
+
```

Comment:
In below case, it will not work as expected:

```graphql
// Block 0
// - miner: AAAA
// - transactions: [tx{...}, tx{...}]

// Block 1
// - miner: BBBB
// - transactions: [tx{...}, tx{...}]

// Block 2
// - miner: AAAA
// - transactions: []

blocks(miner: "AAAA", excludeEmptyTxs: true) {
  index
}

// expected: [{ index: 0 }]
// actual: [{ index: 0 }, { index: 2 }]
```

Reply:
If it was hard to find the solution, could you leave a description for the miner option at least?

Reply:
> If it was hard to find the solution,

I believe it must not be difficult.  Merge two conditionals into one:

```csharp
if ((miner is null || miner == block.Miner) &&
    (!excludeEmptyTxs || block.Transactions.Any()))
```